native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -256,12 +256,43 @@ void EventDispatcher::dispatchResizeEvent(int width, int height) {
         jsResizeEventObj = se::Object::createPlainObject();
         jsResizeEventObj->root();
     }
+//    reinterpret_cast<BaseGame>(CC_CURRENT_APPLICATION_SAFE())->_windowInfo.width = width;
+//    reinterpret_cast<BaseGame>(CC_CURRENT_APPLICATION_SAFE())->_windowInfo.height = height;
 
     jsResizeEventObj->setProperty("width", se::Value(width));
     jsResizeEventObj->setProperty("height", se::Value(height));
+    
     se::ValueArray args;
     args.emplace_back(se::Value(jsResizeEventObj));
-    EventDispatcher::doDispatchEvent(EVENT_RESIZE, "onResize", args);
+    const char *jsFunctionName = "onResize";
+    //EventDispatcher::doDispatchEvent(EVENT_RESIZE, "onResize", args);
+
+    if (!se::ScriptEngine::getInstance()->isValid()) {
+        return;
+    }
```

Comment:
no need here, as
- these codes will be invoked again in following codes
- can dispatch events to c++ event script engine is not ready

Reply:
I've modified the logic of all, no need to consider it now